Dataframe.lua

```diff
@@ -119,15 +125,26 @@ end
 -- Internal function to collect columns names
 function Dataframe:_refresh_metadata()
 	keyset={}
-	n=0
-
+  rows = -1
 	for k,v in pairs(self.dataset) do
-		n=n+1
-		keyset[n]=k
+		table.insert(keyset, k)
+
+		local no_rows_in_v = 1
+		if (type(v) == 'table') then
+			no_rows_in_v = table.maxn(v)
+		end
+		if (rows == -1) then
+			rows = no_rows_in_v
+		else
+		 	assert(rows == no_rows_in_v,
```

Comment:
This assert will never be pass with the one-valued column functionality there is in load_table